tests/ui/EditTaskHelpers.test.ts

```diff
@@ -49,31 +49,25 @@ describe('labelContentWithAccessKey() tests', () => {
 });
 
 describe('parseAndValidateRecurrence() tests', () => {
-    const emptyTask = new EditableTask({
-        description: '',
-        status: Status.TODO,
-        priority: 'none',
-        recurrenceRule: '',
-        createdDate: '',
-        startDate: '',
-        scheduledDate: '',
-        dueDate: '',
-        doneDate: '',
-        cancelledDate: '',
-        forwardOnly: true,
-        blockedBy: [],
-        blocking: [],
-    });
-    const noRecurrenceRule: Partial<EditableTask> = emptyTask;
-    const invalidRecurrenceRule: Partial<EditableTask> = {
-        recurrenceRule: 'thisIsWrong',
+    const emptyTask = new TaskBuilder().build();
```

Comment:
I suspect that this actually a change in test behaviour, as `TaskBuilder` has a default description value...
I don't know if you care, but maybe line 52 could set `description` to empty string, just for clarity/consistency.

Reply:
Yay for code review =)

I realise now that this should be a bit different, will make some changes to show the real intent (‘task without recurrence’ probably)

Reply:
Ok, fixed. The focus here is not on the task emptiness, but on the absence of the `Recurrence`